tests/Unit/WaitsForElementsTest.php

```diff
@@ -391,16 +442,19 @@ public function test_wait_for_an_element_to_be_disabled()
     public function test_wait_for_text_in()
     {
         $element = m::mock(RemoteWebElement::class);
-        $element->shouldReceive('getText')->andReturn('Discount: 20%');
+        $element->shouldReceive('getText')->once()->andReturn('Discount: 20%');
 
         $driver = m::mock(WebDriver::class);
+//        $driver->shouldReceive('wait')->with(5, 100)->andReturnUsing(function ($seconds, $interval) use ($driver) {
```

Comment:
I found wait() was sometimes caled with 5 and not 2, and had to use this commented out version. Could not understand why.
Just realized it's probably because of  `test_default_wait_time()` test higher up, setting static value that persists within the class (`Browser::$waitSeconds = 2;`), but sometimes not executed - when filtering only this test for example.

It should probably be reset, either in that test (`try ... finally{Browser::$waitSeconds = 5;}`) or in the class tearDown() method